exercises/concept/chessboard/.docs/instructions.md

```diff
@@ -2,7 +2,21 @@
 
 As a chess enthusiast, you would like to write your own version of the game. Yes, there may be plenty of implementations of chess available online already, but yours will be unique!
 
-Each square of the chessboard is identified by a letter-number pair. The vertical columns of squares, called files, are numbered 1 through 8. The horizontal rows of squares, called ranks, are numbered 1 to 8.
+Each square of the chessboard is identified by a letter-number pair:
+ - The vertical columns of squares, called files, are numbered 1 through 8.
+ - The horizontal rows of squares, called ranks, are numbered 1 to 8.
+
+|   | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 |   |
```

Comment:
I would recommend wrapping the chessboard in a code block so the characters are shown with a monospace font. Otherwise the layout could be off.

Reply:
I can change to a code block. By this you mean tripple ` character?

Reply:
I left you a note how tables look on the website in the new PR. For the code block you only need the 3 backticks, you don't  need additional indentation unless you want the code block to be indented (which is not needed here I think).